sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp

```diff
@@ -3734,11 +3735,11 @@ namespace Azure { namespace Storage { namespace Blobs {
               = BlobLeaseStatusFromString(httpResponse.GetHeaders().at("x-ms-lease-status"));
           response.LeaseState
               = BlobLeaseStateFromString(httpResponse.GetHeaders().at("x-ms-lease-state"));
-          auto response_lease_duration_iterator
```

Comment:
nit: is this change relevant to the PR?

Reply:
Yep, it's to fix a build error caused by this PR. So it's definitely relevant to this PR.

Reply:
How come a change in this PR affects absolutely unrelated code?